src/rmw.c

```diff
@@ -512,6 +509,9 @@ rmw_option_init (rmw_options *options)
   options->want_restore = false;
   options->want_purge = false;
   options->want_empty_trash = false;
+  options->want_orphan_chk = false;
+  options->want_selection_menu = false;
+  options->want_undo = false;
```

Comment:
You could stick "force", "list", and "alt_config"in there as well, removing them from the top of rmw.c